docusaurus.config.js

```diff
@@ -211,6 +211,12 @@ module.exports = {
         theme: {
           customCss: require.resolve('./src/css/custom.css'),
         },
+        // sitemap: {
```

Comment:
Why is this commented out?

Reply:
Ops... Cross-branch contamination 😆 It was not supposed to be there in the first place. Fixed that https://github.com/KILTprotocol/docs/pull/233/commits/4f805f411812d9347a4ccdc4a85633cd43db0a28.